Domain/Search/ShortlistTuitionPartners/TuitionPartner.cs

```diff
@@ -0,0 +1,3 @@
+namespace Domain.Search.ShortlistTuitionPartners;
+
+public record TuitionPartner(string SeoUrl, string? LocalAuthority);
```

Comment:
I wonder if this should be something like `ShortlistedTuitionPartner` or something like that, so that it doesn't get confused with an actual `TuitionPartner` record?